tools/win_build_rdiffbackup.sh

```diff
@@ -18,26 +18,18 @@ else
 fi
 
 PYEXE=python.exe
-PYINST=PyInstaller.exe
+TOXEXE=tox.exe
 if [[ -n ${PYDIRECT} ]]
 then
 	py_dir=C:/Python${bits}
 	PYEXE=${py_dir}/${PYEXE}
-	PYINST=${py_dir}/Scripts/${PYINST}
+	TOXEXE=${py_dir}/Scripts/${TOXEXE}
 fi
 
 LIBRSYNC_DIR=${HOME}/librsync${bits}
 export LIBRSYNC_DIR
 
-ver_name=rdiff-backup-$(${PYEXE} setup.py --version)
-py_ver_brief=${PYTHON_VERSION%.[0-9]}
+RDIFF_BACKUP_VERSION=rdiff-backup-$(${PYEXE} setup.py --version)-${bits}
```

Comment:
Would it be possible to move that into tox.ini ? So that someone that want to build a exe, could simply call `tox -e buildexe`

Reply:
This is a nice to have.

Reply:
I wouldn't know if it's possible in tox.ini... I couldn't find anything at https://tox.wiki/en/latest/config.html